shared/cfg/machines.cfg

```diff
@@ -44,8 +44,8 @@ variants:
         machine_type = arm64-mmio:virt
         # TODO: Change to "gic-version=max" when supported
         machine_type_extra_params = gic-version=host
-        # No support for VGA yet
-        vga = none
+        # Only virtio-gpu-pci supported for now
+        vga = virtio
```

Comment:
Hi @ldoktor I have two questions about the change.

1) Is this a PCI device or mmio device? Your commit message suggests this is a mmio device, but your comment in code says that it's a PCI device? Seems that the "virtio-gpu-pci" in comment should be changed to "vrito-gpu-device"?

2) Is this a virtio-gpu device or virtio-vga device? Your comment in the code suggests it's a virtio-gpu device, but from the existing code in add_vga() in qemu_vm.py, it seems that virtio-vga is actually used? BTW, does virtio-vga works on arm64? I didn't try it because virtio-vga is not compiled into qemu binary on arm64 by default (a search in the makefiles in default-configs dir in QEMU source code shows that it's only available on hppa, ppc64, and i386).

Reply:
Sorry, just saw your change to add_vga(). Please ignore above two questions. My new comments below:

1) Typo in the comment. It should be "virtio-gpu-device"

2) The code uses virtio-gpu-device, but the device is not compiled into qemu binary on aarch64 by default.